### DISCUSSION TITLE
Get id off current display

### DISCUSSION THREAD

**Discussion by jcpoliveira** [asmvik/yabai#829] [Q&A] [answered]
Hi there,

is there a way to query what is the current focused display? 

At the moment Im using:

    yabai -m query --windows --window|jq .display
 
But this only works if there is at least one window.
Thanks!

**Comment by rockyzhang24** [comment, marked as answer]
`yabai -m query --displays --display`
Furthermore, refer to https://github.com/koekeishiya/yabai/blob/master/doc/yabai.asciidoc#query

**Reply by jcpoliveira** [reply]
Thanks for the help. I did glance over this but I incorrectly assumed  [<DISPLAY_SEL>] was mandatory, silly me.
